src/test/java/io/jenkins/plugins/pipeline/cache/CacheStepTest.java

```diff
@@ -403,6 +403,34 @@ public void testDefaultExcludesIgnoredWhenFilterIsActive() throws Exception {
         j.assertBuildStatusSuccess(b);
     }
 
+    @Test
+    public void testDeclarativePipeline() throws Exception {
```

Comment:
I suggest to also add an example to the README so users can quickly see how to use it.

Reply:
If we want to add support for [options](https://www.jenkins.io/doc/book/pipeline/syntax/#options) or [stage-options](https://www.jenkins.io/doc/book/pipeline/syntax/#stage-options), then I would agree.

My experience with declarative pipelines was most of the time that something was not working and then switched to scripted pipelines.